the_wizz/pair_maker.py

```diff
@@ -99,22 +103,23 @@ def run(self, reference_catalog, unknown_catalog, use_unkn_weights=False):
         Parameters
         ----------
         reference_catalog : 'dict' of `numpy.ndarray`
-            Dictionary containing arrays 
+            Dictionary containing arrays
         """
-        unkn_vects = self._convert_radec_to_xyz(
-            np.radians(unknown_catalog["ra"]),
-            np.radians(unknown_catalog["dec"]))
-        unkn_tree = cKDTree(unkn_vects)
+        unkn_tree = SphericalKDTree(
```

Comment:
Either input these as radians or add a bool to the SphereicalKDTree init that allows it to accept angles in degrees at this stage.

Reply:
We decided to only accept radians.